src/app/db/init_db.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from app.api import crud
+from app.db import accesses, users
+from app.api.schemas import AccessCreation, UserCreation
+from app.api.security import hash_password
+
+
+async def init_db():
+
+    login = os.getenv("SUPERUSER_LOGIN")
```

Comment:
Just moving this and `os.getenv("SUPERUSER_PWD")` to a config variable, and I think we are good to go :ok_hand:

Reply:
done !